docs/source/config/details.rst

```diff
@@ -2,89 +2,85 @@
 Specific config details
 =======================
 
+.. _custom_prompts:
+
+Custom Prompts
+==============
+
+.. versionchanged:: 5.0
+
+From IPython 5, prompts are produced as a list of Pygments tokens, which are
+tuples of (token_type, text). You can customise prompts by writing a method
+which generates a list of tokens.
+
+There are four kinds of prompt:
```

Comment:
Not sure if this should be prompt or prompts. Likely could be either. No need to revise if you are fine with prompt.